biztalk/technical-guides/checklist-optimizing-performance-on-hyper-v.md

```diff
@@ -23,15 +23,15 @@ The following considerations apply when running BizTalk Server and/or the [!INCL
+|                      Scope the hardware requirements for the [!INCLUDE[btsBizTalkServerNoVersion](../includes/btsbiztalkservernoversion-md.md)] solution.                       | -   Follow the guidance in the “Planning the Environment for BizTalk Server” section of the [!INCLUDE[btsBizTalkServerNoVersion](../includes/btsbiztalkservernoversion-md.md)] Operations Guide [here](https://go.microsoft.com/fwlink/?LinkId=122399) to scope the hardware requirements for the solution.<br />-   To scope the edition and number of BizTalk Servers that will be required by the solution review BizTalk Server planning considerations documented in “Planning the BizTalk Server Tier” [here](https://go.microsoft.com/fwlink/?LinkId=122401).<br />-   To scope the version and number of [!INCLUDE[btsSQLServerNoVersion](../includes/btssqlservernoversion-md.md)] computers that will be required by the solution, review database planning considerations documented in “Planning the Database Tier” [here](https://go.microsoft.com/fwlink/?LinkId=122402) and the “Performance Overhead of Running SQL Server in Hyper-V” section of the “Running SQL Server 2008 in a Hyper-V Environment Best Practices and Performance Considerations” whitepaper available [here](https://go.microsoft.com/fwlink/?LinkId=144622).<br />-   To complete planning for development, testing, staging, and production environments review “Planning the Development, Testing, Staging, and Production Environments” [here](https://go.microsoft.com/fwlink/?LinkId=122403). |
```

Comment:
Fixed "[here]" link text, changed curly quotes to plain quotes, and fixed table formatting for easier readability:

```suggestion
| Scope the hardware requirements for the [!INCLUDE[btsBizTalkServerNoVersion](../includes/btsbiztalkservernoversion-md.md)] solution. | - To scope the hardware requirements for the solution, follow the guidance in the "Planning the Environment for BizTalk Server" section in the [!INCLUDE[btsBizTalkServerNoVersion](../includes/btsbiztalkservernoversion-md.md)] [Operations Guide](https://go.microsoft.com/fwlink/?LinkId=122399). <br><br>- To scope the edition and number of BizTalk Servers that will be required by the solution review BizTalk Server planning considerations documented in "Planning the BizTalk Server Tier" [here](https://go.microsoft.com/fwlink/?LinkId=122401). <br><br>- To scope the version and number of [!INCLUDE[btsSQLServerNoVersion](../includes/btssqlservernoversion-md.md)] computers that the solution requires, review the database planning considerations in the following documentation: <br>--- ["Planning the Database Tier"](https://go.microsoft.com/fwlink/?LinkId=122402) <br>--- The "Performance Overhead of Running SQL Server in Hyper-V" section in the ["Running SQL Server 2008 in a Hyper-V Environment Best Practices and Performance Considerations" whitepaper](https://go.microsoft.com/fwlink/?LinkId=144622). <br><br>- To complete planning for development, testing, staging, and production environments, review ["Planning the Development, Testing, Staging, and Production Environments"](https://go.microsoft.com/fwlink/?LinkId=122403). |
```